Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -316,7 +341,7 @@ function QuestieOptions.tabs.general:Initialize()
             --Spacer_B = QuestieOptionsUtils:Spacer(1.73),
             questannounce = {
                 type = "select",
-                order = 9,
+                order = 11,
```

Comment:
Not sure if this intended, but the order now is 9 -> 8 -> 11. Guess you oversaw the `acceptTrivial` option.